linux/src/threadpool_linux.c

```diff
@@ -157,32 +160,35 @@ static int threadpool_work_func(void* param)
 
         do
         {
+            /* Codes_SRS_THREADPOOL_LINUX_11_056: [ threadpool_work_func shall get the real time by calling clock_gettime. ]*/
             if (clock_gettime(CLOCK_REALTIME, &ts) == -1)
             {
-                // Consider Log
+                /* Codes_SRS_THREADPOOL_LINUX_11_080: [ If clock_gettime fails, threadpool_work_func shall return. ]*/
```

Comment:
>return [](http://example.com/codeflow?start=107&length=6)

doesn't seem like that's what happens, the loop will run this again.

Reply:
done